cypress/e2e/PageObjects/dashboardPO.cy.js

```diff
@@ -166,21 +167,67 @@ function CloseModalButton() {
   return cy.get("[data-cy ='x-button']")
 }
 
-function validateExitBetaModalbuttonLink() {
+function getcardNumber() {
+  var a = []
```

Comment:
`var` should be replaced with `let`

If the variable will never be reassigned or modified, `const` should be used :)

Source:

https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994

Reply:
> `var` should be replaced with `let`
> 
> If the variable will never be reassigned or modified, `const` should be used :)
> 
> Source:
> 
> https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994

Thanks,just changed it to const  :)